### ISSUE TITLE
Deposited badge on work show page does not meet color contrast accessibility requirements

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6755]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.0), the blue `Deposited` badge does not meet WCAG AA minimum color contrast requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2.  Go to https://dev.nurax.samvera.org/concern/generic_works/x920fx181?locale=en run the Site Improve browser extension

![color_contrast_deposited](https://github.com/samvera/hyrax/assets/32644254/2dd5464c-0a80-41bc-8d59-6f9f1ae42877)

### Rationale (for feature request only)

Hyrax (and nurax) should be accessible to users out of the box.
